tests/unit/workflows/test_workflows_tasks_upload.py

```diff
@@ -72,7 +75,8 @@ def test_set_schema_builds_a_full_url_for_the_schema():
 
     data = {'$schema': 'hep.json'}
     extra_data = {}
-    assert validate(data['$schema'], subschema) is None
+    with pytest.raises(ValidationError):
+        validate(data['$schema'], subschema) is None
```

Comment:
This is notable: `inspire-dojson` puts something invalid there, which `set_schema` makes valid. This is the only exception to the usual task invariant (valid in, valid out).